src/neuroglancer/ui/annotations.ts

```diff
@@ -1132,7 +1132,7 @@ export class AnnotationLayerView extends Tab {
   }
 
   private exportToCSV() {
-    const filename = 'annotations.csv';
```

Comment:
Using layer name as a default is fine, but from a user perspective its very obtuse. Even I didn't understand what was going on at first. 

ExportCSV shouldn't be used so frequently that a dialog box to name the CSV would be much of an interruption and its much more apparent. 

For power users, maybe have a keyboard shortcut as a compromise.